src/sys/socket/addr.rs

```diff
@@ -64,6 +64,24 @@ impl InetAddr {
         }
     }
 
+    /// Creates an `IpAddr` struct from libc's sockaddr.
```

Comment:
Right, that's a last-minute typo. I didn't notice `SockAddr` before and currently think the `getifaddrs` API shoud rather use `SockAddr`.